server/proxy/bindings.go

```diff
@@ -136,11 +136,6 @@ func (p *Proxy) scanAppBindings(app *apps.App, bindings []*apps.Binding, locPref
 			continue
 		}
 
-		if locPrefix == apps.LocationCommand {
```

Comment:
@hanzei Can we still default to AppID if the location/label are missing, or are you >0/5 on encouraging hierarchical AppIDs?

Reply:
I'm opposed to using the AppID by default for two reasons:
1. 3/5 I would like to think about the AppID as "a technical detail". It's a technical necessity and should not be user facing.
2. 1/5 Default values like that are not obvious. I would like to rather error with a meaningful message to developer then doing "magic".

Reply:
SG